file-store/src/lib.rs

```diff
@@ -58,6 +58,16 @@ impl FileStore {
 
         Ok((uuid, update_file))
     }
+    /// Creates a new temporary update file with the given Uuid.
+    /// A call to `persist` is needed to persist the file in the database.
+    pub fn new_update_woth_uuid(&self, uuid: u128) -> Result<(Uuid, File)> {
```

Comment:
```suggestion
    pub fn new_update_with_uuid(&self, uuid: u128) -> Result<(Uuid, File)> {
```
Sorry :smiling_face_with_tear: